src/lang/gettext_string_hash.hpp

```diff
@@ -0,0 +1,69 @@
+//
+// Created by machaviprace on 22.07.21.
+//
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+
+#define HASHWORDBITS 32
+
+//template <uint32_t TEST>
+class CTracker {
+public:
+    CTracker(volatile int32_t *duration);
+    ~CTracker();
+
+private:
+    volatile int32_t *Duration;
+    uint32_t start = 0;
+};
+
+class gettext_hash_table {
+public:
+    gettext_hash_table() = default;
+    /**
```

Comment:
ideally please use the more compact `///` variant of doxygen comments - it is much easier to type and read